### DISCUSSION TITLE
Webkit in CSS3

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#54] [Q&A] [answered]
What is the use of Webkit in CSS3?

**Comment by Bunny77K9** [comment, marked as answer]
Webkit is an important software component in CSS that allows for the easy rendering of HTML and CSS elements in a variety of browsers, such as Chrome, Firefox, and Safari.

There are many engines for browsers such as:

    Gecko for Mozilla
    Presto for Opera
    Edge for Internet Explorer
